fitsio/fitslib.py

```diff
@@ -345,6 +347,11 @@ def write(filename, data, extname=None, extver=None, header=None,
         (case-insensitive) See the cfitsio manual for details.
     tile_dims: tuple of ints, optional
         The size of the tiles used to compress images.
+    qlevel: float, optional
+        Quantization level.  Lower generally result in more compression, we
+        recommend one reads the FITS standard or cfitsio manual to fully
+        understand the effects of quantization.  None or 0 means no
+        quantization, and for gzip also implies lossless.  Default is 4.0
```

Comment:
I think the `qlevel` documentation (which unfortunately must be repeated a lot) should mention that this is only relevant for floating-point data.